libs/ngrx-toolkit/src/lib/devtools/internal/devtools-syncer.service.ts

```diff
@@ -54,21 +54,47 @@ export class DevtoolsSyncer implements OnDestroy {
   #currentId = 1;
 
   readonly #connection: Connection = this.#isBrowser
-    ? window.__REDUX_DEVTOOLS_EXTENSION__
-      ? window.__REDUX_DEVTOOLS_EXTENSION__.connect(this.#devtoolsConfig)
-      : dummyConnection
+    ? this.#initDevtoolsConnection()
     : dummyConnection;
 
   constructor() {
     if (!this.#isBrowser) {
-      return;
+      console.warn(
```

Comment:
No warnings if we are on the server. We had
this before and users were complaining that
we are filling up their logs.